cmd/podman/machine/init.go

```diff
@@ -111,10 +111,12 @@ func initMachine(cmd *cobra.Command, args []string) error {
 		vm  machine.VM
 		err error
 	)
-
 	provider := getSystemDefaultProvider()
 	initOpts.Name = defaultMachineName
 	if len(args) > 0 {
+		if len(args[0]) > 30 {
```

Comment:
Could you turn `30` into a const with a comment?  I am certain there is some important reasoning behind that may be worth preserving.